filters-share/extended_css_ELEMHIDE.txt

```diff
@@ -1,3 +1,4 @@
+namu.wiki#?#article div:has( div.namuwiki-toc-ad ) ~ div:has( div.wiki-paragraph ):has(table > tbody > tr > td[colspan] > div > img[src*="w.namu.la/"]):not(:has( div.namuwiki-toc-ad ))
```

Comment:
It's reasonable to remove the part: `div:has( div.wiki-paragraph )` for performance as almost every div in namuwiki has empty `.wiki-paragraph` node.

Reply:
Cloud you give me a suggestion?

Reply:
Did you mean this?
```suggestion
namu.wiki#?#article div ~ div:has( div.wiki-paragraph ):has(table > tbody > tr > td[colspan] > div > img[src*="w.namu.la/"]):not(:has( div.namuwiki-toc-ad ))
```

Reply:
Ah not that one. `.namuwiki-toc-ad` is required. I am building one. `div.wiki-paragraph` is meaningless as they update the site structure.

Reply:
Based on my rule: `namu.wiki#?#article div:has(a[href*="/w/%EB%B6%84%EB%A5%98:"]) ~ div:has(td img[src^="//ww.namu.la/s/"][class]):not(:has(div.namuwiki-toc-ad, a[href^="#s-"])) table`

You may change the rule as possible.

Reply:
@piquark6046 I need cross-check if the rule is working for both side. I don't know how many versions that namuwiki is distributing so far but yours isn't working on my side.

`article div:has( div.namuwiki-toc-ad ) ~ div:has( div.wiki-paragraph ):has(table > tbody > tr > td[colspan] > div > img[src*="w.namu.la/"]):not(:has( div.namuwiki-toc-ad ))`

Reply:
https://github.com/List-KR/List-KR/pull/261#discussion_r834090968
This rule does not work on my site.

Reply:
Confirmed.